.gitignore

```diff
@@ -130,3 +133,4 @@ dmypy.json
 
 # data
 raw_data
+data
```

Comment:
I added this because that is where the tutorial and example notebooks often look. We could also rename all of those hardcoded paths to "raw_data" and not add a "data" field here.